packages/react-native/ReactCommon/react/renderer/mapbuffer/tests/MapBufferTest.cpp

```diff
@@ -27,6 +28,22 @@ TEST(MapBufferTest, testSimpleIntMap) {
   EXPECT_EQ(map.getInt(1), 4321);
 }
 
+TEST(MapBufferTest, testSimpleLongMap) {
+  auto builder = MapBufferBuilder();
+
+  int64_t minInt64 = std::numeric_limits<int64_t>::min();
+  int64_t maxInt64 = std::numeric_limits<int64_t>::max();
+
+  builder.putLong(0, minInt64);
```

Comment:
Could we add tests for more interesting cases? Like large (>52 bit) numbers, negative numbers?